content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -15,6 +15,17 @@ This document serves both as a reference to the values and as a coordination poi
 
 ## Labels, annotations and taints used on API objects
 
+
+### apf.kubernetes.io/autoupdate-spec
+
+Type: Annotation
+
+Example: `apf.kubernetes.io/autoupdate-spec: "true"`
+
+Used on: [Mandatory and suggested APF configuration objects](/concepts/cluster-administration/flow-control/#defaults)
```

Comment:
There are only two kinds defined in the flow control API group, `FlowSchema` and `PriorityLevelConfiguration`. Is it much clearer if we explicitly say that this annotation is to be applied on such objects?